tests/common/test_cgroupapi.py

```diff
@@ -410,8 +416,10 @@ def mock_popen(*args, **kwargs):
 
                     self.assert_cgroups_created(extension_cgroups)
 
-    @skip_if_predicate_false(i_am_root, "Test does not run when normal user")
+    @attr('requires_sudo')
```

Comment:
this test actually does no require root privileges

Reply:
You're right, updated.